eng/common/scripts/Update-ChangeLog.ps1

```diff
@@ -12,14 +12,13 @@ param (
   [String]$ServiceDirectory,
   [Parameter(Mandatory = $true)]
   [String]$PackageName,
-  [String]$Unreleased = "true", #Argument is string becasue of the different ways the script is called in the various repos.
-  [String]$ReplaceLatestEntryTitle = "false",
+  [Parameter(Mandatory = $false)]
+  [Switch]$Unreleased = $true,
```

Comment:
You generally don't want to default switch parameters to true otherwise in order to ever set it to false someone has to pass "-Unreleased:false" and we lose most of the benefits of using a switch. You might have to change the name slightly to make it make since, perhaps "$Release=$false" or something like that.

Reply:
I just went ahead and changed it to a Boolean.